crates/db_schema/src/utils.rs

```diff
@@ -272,13 +272,15 @@ static EMAIL_REGEX: Lazy<Regex> = Lazy::new(|| {
 });
 
 pub mod functions {
-  use diesel::sql_types::{BigInt, Text, Timestamp};
+  use diesel::sql_types::{BigInt, Nullable, Text, Timestamp};
 
   sql_function! {
     fn hot_rank(score: BigInt, time: Timestamp) -> Integer;
   }
 
   sql_function!(fn lower(x: Text) -> Text);
+
+  sql_function! { fn coalesce_time(x: Nullable<Timestamp>, y: Timestamp) -> Timestamp; }
```

Comment:
I don't think that's a function. Should just be `coalesce` . 

I see there's another coalesce function in `post_view.rs`, using the different types. If its not possible to make them generic, we could just scope them differently.

Reply:
Hmm that makes sense if it picks the SQL function by Rust function name.